lib/__tests__/cookieJar.spec.ts

```diff
@@ -1247,6 +1248,9 @@ it('should fix issue #154 - Expiry should not be affected by creation date', asy
   ])
   // the expiry time should be 60s from now (1000)
   expect(updatedCookies[0]?.expiryTime()).toBe(now + 60 * 1000 + 1000)
+  expect(updatedCookies[0]?.expiryDate()).toEqual(
+    new Date(now + 60 * 1000 + 1000),
+  )
```

Comment:
At some point we should probably go through and ensure we actually assert the existence of everything we expect to have. 😅

Reply:
Yeah, we also were operating under the assumption that the vows test suite would be our safety net to prevent us from making incorrect changes to the public API. As we've seen, that can be misleading if there is no coverage for a method.